examples/resources/snowflake_database_old/import.sh

```diff
@@ -0,0 +1 @@
+terraform import snowflake_database_old.example name
```

Comment:
Because they're two different types. if you would like to import the state into `resource "snowflake_database_old"` you cannot use `terraform import snowflake_database`

Reply:
xD

I meant name and `'` :D